server.ts

```diff
@@ -1,11 +1,21 @@
-import path from "path";
+import fs from "node:fs";
+import path from "node:path";
 
 import prom from "@isaacs/express-prometheus-middleware";
 import { createRequestHandler } from "@remix-run/express";
+import { broadcastDevReady, installGlobals } from "@remix-run/node";
+import chokidar from "chokidar";
 import compression from "compression";
+import type { RequestHandler } from "express";
 import express from "express";
 import morgan from "morgan";
 
+installGlobals();
+
+const BUILD_PATH = path.join(process.cwd(), "build", "index.js");
+/** @type { import('@remix-run/node').ServerBuild | Promise<import('@remix-run/node').ServerBuild> } */
+let build = await import(BUILD_PATH);
```

Comment:
Top level await is failing with following TS error
![](https://github.com/remix-run/blues-stack/assets/6643991/2ac8f4d3-0764-4f03-a92a-41f66f8c388f)

@pcattori Not sure what the right way would be as we're still officially supporting Node 14

Reply:
for now I just wrapped this in a `async function run() {...}`